src/Spectre.Database/Entities/Enumerations.cs

```diff
@@ -0,0 +1,78 @@
+﻿/*
+ * Enumerations.cs
+ * File providing enumerations for entity classes.
+ *
+   Copyright 2017 Roman Lisak
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Spectre.Database.Entities
```

Comment:
Spectre.Database.Entities.Enums will make it easier for other team members to find enums declared here when they will need to develop something DB related ;)